test/integrations/routes/comments.test.js

```diff
@@ -107,6 +108,100 @@ describe('API endpoint: /api/articles/:slug/comments (Routes)', () => {
       });
   });
 
+  it('Should update a comment2', (done) => {
```

Comment:
#### Critical
- I honestly can't make out what `comment3`, `comment4`  and `comment5` are about? Can you be more descriptive of what you are testing.